identity/app/model/FrontendSavedArticles.scala

```diff
@@ -2,16 +2,16 @@ package model
 
 import com.gu.identity.model.{SavedArticle, SavedArticles}
 import org.joda.time.DateTime
-import org.joda.time.format.ISODateTimeFormat
+import org.joda.time.format.{DateTimeFormat, ISODateTimeFormat}
 
 /**
  * Created by nbennett on 27/03/15.
  */
-class FrontendSavedArticles(version: String, articles: List[SavedArticle]) extends SavedArticles(version, articles) {
+class FrontendSavedArticles(version: String, articles: List[FrontendSavedArticle]) extends SavedArticles(version, articles) {
```

Comment:
I'm not sure about the value of `FrontendSavedArticle`, see my comment in the PR